src/test/scala/org/scalastyle/scalariform/ImportsCheckerTest.scala

```diff
@@ -177,3 +177,63 @@ object Barbar {
     assertErrors(List(), source)
   }
 }
+
+class ImportOrderCheckerTest extends AssertionsForJUnit with CheckerTest {
+  val key = "import.ordering"
+  val classUnderTest = classOf[ImportOrderChecker]
+
+  @Test def testImportGrouping(): Unit = {
+    val source = """
+      |package foobar
+      |
+      |import java.util.Map
```

Comment:
A lot of logic is around treating wildcard imports, it would be good to also test that behaviour here.